lib/ansible/modules/web_infrastructure/jira.py

```diff
@@ -390,6 +397,7 @@ def main():
             linktype=dict(),
             inwardissue=dict(),
             outwardissue=dict(),
+            timeout=dict(type='float', default=10),
```

Comment:
it is a float, I just wondered if it would make any difference from user perspective. But it AFAICS not. LGTM